flink-end-to-end-tests/run-nightly-tests.sh

```diff
@@ -235,15 +235,15 @@ printf "Running Java end-to-end tests\n"
 printf "==============================================================================\n"
 
 
-LOG4J_PROPERTIES=${END_TO_END_DIR}/../tools/log4j-travis.properties
+LOG4J_PROPERTIES=${END_TO_END_DIR}/../tools/ci/log4j-ci.properties
```

Comment:
nit: the "ci" suffix could probably be removed, given that the directory already contains "ci".